src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -338,6 +338,21 @@ private String filterForSitesWithOwnerRole(String projectsString, String current
 			// filter out those sites that current user has role "Owner" in it
 			for (int iSite = 0; sitesJSONArray != null && iSite < sitesJSONArray.length(); iSite++) {
 				JSONObject siteJSON = sitesJSONArray.getJSONObject(iSite);
+				
+				// check for site id
+				if (siteJSON.isNull("id"))
```

Comment:
Looks ok.  But if there are any other isNull checks consider making a predicate method that will do the check and any required logging and then check the list of required attributes in a single if  / continue statement.